src/ItemList.php

```diff
@@ -175,6 +175,15 @@ public function hasConstrainedItems(): bool
         return $this->hasConstrainedItems;
     }
 
+    public function getTotalVolume(): int
+    {
+        $volume = 0;
+        foreach ($this->list as $item) {
+            $volume += $item->getDepth() * $item->getWeight() * $item->getLength();
```

Comment:
I think you meant `* $item->getWidth()` 😉 . Don't forget to add a test.

(width, length, depth) is the ordering I try and stick to

Reply:
Ahh, good catch! I removed the method since I see you've done something similar in the new `getBoxList` method to stay consistent. Let me know if you'd prefer it to be added back to reduce code duplication.